### PR TITLE
Restaurant class

### PR DESCRIPTION
**Pull request by HelenKoppel**
Please review

### CODE REVIEW

File: src/main/java/model/Restaurant.java [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+@Entity
 public class Restaurant {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int restaurant;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "address")
+    private String address;
+
```
Comment (kristjanhollo) [inline src/main/java/model/Restaurant.java:14]: Getter & Setters missing
@Column missing (restaurant_id)

**Comment by HelenKoppel** [comment]
Please review

**Comment by HelenKoppel** [comment]
Join done

**Review by Rix1113 — COMMENTED** [review]
**unnecessary**  
@ManyToOne
    @JoinColumn(name = "restaurantId")
    private Restaurant restaurant;

Add relationship between Restaurant and Menu

**Review by kristjanhollo — APPROVED** [review]

**Pull request merged**